sdktests/server_side_persistence_consul.go

```diff
@@ -0,0 +1,121 @@
+package sdktests
+
+import (
+	"fmt"
+	"strings"
+
+	consul "github.com/hashicorp/consul/api"
+	"github.com/launchdarkly/sdk-test-harness/v2/servicedef"
+)
+
+type ConsulPersistentStore struct {
+	consul *consul.Client
+}
+
+// {{{ PersistentStore implementation
```

Comment:
Is this a fold marker? Is it intentional?

Reply:
Well yes and no. I was wanting to show where it started and ended, so I was putting in some easy markers that you can jump between.

Are they also default vim markers? Yes, but that's beside the point. 😄 

Honestly, I was expecting more work in these files and figured it would be more useful than it is. So I will remove it.